src/safir/logging.py

```diff
@@ -4,78 +4,27 @@
 
 import logging
 import sys
-from contextvars import ContextVar
 from typing import TYPE_CHECKING
 
 import structlog
 
-__all__ = ["configure_logging", "response_logger", "get_response_logger"]
-
-
 if TYPE_CHECKING:
-    from typing import Any, List
-
-    from structlog._config import BoundLoggerLazyProxy
+    from typing import Any, List, Optional
 
+__all__ = ["configure_logging", "logger_name"]
 
-response_logger: ContextVar[BoundLoggerLazyProxy] = ContextVar(
-    "response_logger"
-)
-"""A context-local structlog logger.
+logger_name: Optional[str] = None
+"""Name of the configured global logger.
 
-This logger is set by the `safir.middleware.bind_logger` middleware.
+When :py:func:`configure_logging` is called, the name of the configured logger
```

Comment:
Here's where you can just do single back ticks, like

```suggestion
When `configure_logging` is called, the name of the configured logger
```

(I realize this is my fault because the Developer Guide needs to be updated to cover this)